app/assets/stylesheets/components/common/_contact_panel.scss

```diff
@@ -3,25 +3,43 @@
 // Styleguide Contact Panel
 
 .contact-panel {
+  // N.B. This is to over-ride styles from Yeast
+  // Should be added to that for a final deployment
+  @include column(12);
```

Comment:
The styles for .contact-panel and .l-contact-panel (here and in two places below) should be updated in Yeast. IIRC they were necessary for the grey background to render properly position-wise.

Reply:
As discussed with Jason, this style is OK to leave in frontend - but can we delete the comments?